CoinDock-app/src/Shared/Form/Name/Name.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import propTypes from "prop-types";
+import "Shared/common-styles/space.css";
+import "Shared/common-styles/common.css";
+export const nameValidation = (value, label = "Name", length = 0) => {
+  let error = null;
+
+  if (!value) {
+    error = `${label} is required`;
+  } else if (value.length > 45) {
+    error = ` The ${label} may not be greater than ${length} characters.`;
+  }
+  return error;
+};
+
+const Name = ({
+  label,
+  name,
+  placeholder,
+  formErrors,
+  onInput,
+  currentFieldValue,
+}) => {
+  const initialValues = {
+    firstname: "",
+    lastname: "",
+  };
+  const [formValues, setformValues] = useState(initialValues);
```

Comment:
Here formvalues is not necessary. as it not being used any where.
it is reseting from the setFormvalues and used set in the setFormValues.
So, I think we can remove it.

Reply:
Removed